modules/specialized-guides/pages/public-cloud-guide/payg/azure/payg-azure-server-setup.adoc

```diff
@@ -0,0 +1,102 @@
+[[azure-server-setup]]
+= Initial setup
+
+.Procedure: Login and update the system
+. SSH into your {azure} instance.
+
+. Switch to the root user and update the system.
++
+
+----
+sudo -i
+zypper refresh
+zypper update
+----
+
+== Network Configuration
+
+Before initiating `yast2 susemanager_setup`, ensure the network is correctly configured for {azure}.
+
+.Procedure: Network Configuration
+. Ensure the network configuration aligns such that `hostname -f` yields the identical name as the reverse DNS lookup of the private IP address. For instance, when executing `nslookup 10.0.0.X`.
+
+. Insert the private IP with its Fully Qualified Domain Name (FQDN) into `/etc/hosts`. For example: +
+`10.0.0.4   instancename.internal.cloudapp.net`
+
+. Edit `/etc/sysconfig/network/config` and append `internal.cloudapp.net` to `NETCONFIG_DNS_STATIC_SEARCHLIST`.
+
+. Execute `netconfig update`.
+
+. Subsequently, `hostname -f` should return the same FQDN as obtained from `nslookup 10.0.0.X`.
+
+[IMPORTANT]
+====
+When adding new virtual machines (VMs), such as {sles} 15 SP5, it's crucial to ensure that they are configured within the same subnet as the SUSE Manager instance. This step is essential for successful network setup and integration.
+
+Additionally, be mindful of the geographical restrictions associated with your plans. Make sure to utilize the appropriate Azure accounts which correspond to the specific geographic zones of your project. 
+====
+
+== Configure Storage
+. Configure storage with the **suma-storage** tool. 
+** For help see: `suma-storage --help`
+This tool simplifies creating the storage disk device and database device.
+
+. Use `lsblk` to see the names of your disk devices.
+You will see a list of partitions that were created when we configured the {azure} image.
+
+** Root partition 200 GB
```

Comment:
100 GB

Reply:
Should also be changed in the AWS specific section to `100GB` for the root partition.